tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -682,3 +682,11 @@ describe('Announcement Dialogs', () => {
     cy.get('[data-cy=announcement-dialog]').should('not.exist');
   });
 });
+
+// Check for canonical link in head
+describe('Canonical Link', () => {
```

Comment:
Let's move the test into `tests/e2e/specs/out-of-game/authentication.spec.js` in the existing describe block for 'page contents'